app/models/claim.rb

```diff
@@ -115,6 +116,8 @@ class Claim < ApplicationRecord
   scope :submitted, -> { where.not(submitted_at: nil) }
   scope :awaiting_checking, -> { submitted.left_outer_joins(:check).where(checks: {claim_id: nil}) }
   scope :approved, -> { joins(:check).where("checks.result" => :approved) }
+  scope :approaching_checking_deadline, -> { awaiting_checking.where("submitted_at < ?", CHECKING_DEADLINE.ago + 2.weeks).where.not("submitted_at < ?", CHECKING_DEADLINE.ago) }
```

Comment:
Could we pull the `+ 2.weeks` out into another constant? What we consider to be "approaching" is a bit hidden.